gcloud/_apitools/transfer.py

```diff
@@ -820,8 +714,11 @@ def InitializeUpload(self, http_request, http=None, client=None):
         if http_response.status_code != http_client.OK:
             raise exceptions.HttpError.FromResponse(http_response)
 
-        self.__server_chunk_granularity = http_response.info.get(
-            'X-Goog-Upload-Chunk-Granularity')
+        # XXX when is this getting converted to an integer?
```

Comment:
These PRs are all part of the work-in-progress to bring the apitools code in:  I fully intend to remove the XXX bits in the subsequent PRs, but they mark points of learning about the grotty parts of the code / protocol, found through hard slogging to get test coverage.